packages/playground/vite.config.ts

```diff
@@ -12,6 +12,14 @@ const enableIstanbul = !!process.env.CI || !!process.env.COVERAGE;
 
 // https://vitejs.dev/config/
 export default defineConfig({
+  define: {
+    'import.meta.env.PLAYGROUND_SERVER': JSON.stringify(
+      process.env.PLAYGROUND_SERVER ?? 'http://localhost:8787'
```

Comment:
I didn't see the production URL configured, is it using localhost in production?

Reply:
No, the production URL is configured at the Vercel site. It has been tested and works well.